ecs/daemon/inputs/host/all-host.conf

```diff
@@ -0,0 +1,65 @@
+[INPUT]
+    Name                tail
+    # Tag is set with underscores for cloud init because s3 output option
+    # s3_key_format_tag_delimiters includes dash
+    Tag                 host.var.log.cloud_init
+    Path                /var/log/cloud-init.log*
+    DB                  /var/fluent-bit/state/flb_cloud_init.db
+    Mem_Buf_Limit       5MB
+    Skip_Long_Lines     On
+    Refresh_Interval    10
+    Rotate_Wait         30
+    storage.type        filesystem
+    # cloud init runs on startup before Fluent Bit, so Read_From_Head On
+    # is required to get those logs
+    Read_From_Head      On
+
+[INPUT]
+    Name                tail
+    # Tag is set with underscores for cloud init because S3 output option
+    # s3_key_format_tag_delimiters includes dash
+    Tag                 host.var.log.cloud_init_output
+    Path                /var/log/cloud-init-output.log*
+    DB                  /var/fluent-bit/state/flb_cloud_init_output.db
+    Mem_Buf_Limit       5MB
+    Skip_Long_Lines     On
+    Refresh_Interval    10
+    Rotate_Wait         30
+    storage.type        filesystem
+    # cloud init runs on startup before Fluent Bit, so Read_From_Head On
+    # is required to get those logs
+    Read_From_Head      On
+
+
+[INPUT]
+    Name                tail
+    Tag                 host.*
+    Path                /var/log/secure*
+    Parser              syslog
+    DB                  /var/fluent-bit/state/flb_secure.db
+    Mem_Buf_Limit       5MB
+    Skip_Long_Lines     On
+    Refresh_Interval    10
+    Read_from_Head      ${READ_FROM_HEAD_HOST}
+
+[INPUT]
+    Name                tail
+    Tag                 host.*
+    Path                /var/log/messages*
+    Parser              syslog
+    DB                  /var/fluent-bit/state/flb_messages.db
+    Mem_Buf_Limit       5MB
+    Skip_Long_Lines     On
+    Refresh_Interval    10
+    Read_from_Head      ${READ_FROM_HEAD_HOST}
+
+[INPUT]
+    Name                tail
+    Tag                 host.*
+    Path                /var/log/dmesg*
+    Parser              syslog
```

Comment:
My comment here is applicable: https://github.com/aws-samples/amazon-cloudwatch-container-insights/pull/108#issuecomment-1353804758

Reply:
I think for this daemon service template, putting the dmesg log inside of a key called "log" is all we want... so just removing the parser here.